### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by artem-lukanin**

### CODE REVIEW

**Review by BileyHarryCopter — COMMENTED** [review]
This honest work, but small

**Review by ArtemMaslov — APPROVED** [review]

**Review by Maksimonk — COMMENTED** [review]
Redo it

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # Test2023
-Test project
+Test project for 2023
+
+sdfsdfsdfsdfsdfsdfsd
```
Comment (NikitaDzer) [inline README.md:4]: What is that cringe?

**Review by NikitaDzer — CHANGES_REQUESTED** [review]
Rework it.

**Pull request merged**